libavcodec/vvc/vvc_cabac.c

```diff
@@ -1865,21 +1887,30 @@ static int abs_decode(VVCLocalContext *lc, const int c_rice_param)
             suffix = (suffix << 1) | vvc_get_cabac_bypass(&lc->ep->cc);
         }
     } else {
-        suffix = limited_kth_order_egk_decode(&lc->ep->cc, c_rice_param + 1, 11, 15);
+        suffix = limited_kth_order_egk_decode(&lc->ep->cc,
+                                              c_rice_param + 1,
+                                              26 - sps->log2_transform_range,
+                                              sps->log2_transform_range);
     }
     return suffix + (prefix << c_rice_param);
 }
 
 static int abs_remainder_decode(VVCLocalContext *lc, const ResidualCoding* rc, const int xc, const int yc)
 {
-    const int c_rice_param = abs_get_rice_param(rc, xc, yc, 4);
+    const VVCSPS *sps = lc->fc->ps.sps;
+    const VVCSH *sh = &lc->sc->sh;
+    const int base_level = sps->rrc_rice_extension_flag
```

Comment:
how about 
base_level[][][] = { 
   { {4, 4},  {4, 4},
   { {2, 1},  {3, 2} };

then we use 
base_level[sps->rrc_rice_extension_flag][sps->bit_depth > 12] [sh->slice_type == VVC_SLICE_TYPE_I]